code/script.js

```diff
@@ -323,21 +355,40 @@ const filterCharacters = (keep) => {
 
 // when clicking guess, the player first have to confirm that they want to make a guess.
 const guess = (suspect) => {
+  //console.log(suspect)
+  const isConfirmed = confirm(`Are you sure you want to make a guess on ${suspect}?`)
+  //console.log(isConfirmed)
+  if(isConfirmed) {
+    checkMyGuess(suspect)
+  }
   // store the interaction from the player in a variable.
   // remember the confirm() ?
   // If the player wants to guess, invoke the checkMyGuess function.
 }
 
 // If you confirm, this function is invoked
 const checkMyGuess = (suspect) => {
+  console.log("suspect",suspect)
+  console.log("secret person", secret.name)
+  if(suspect === secret.name)
+    //{console.log(`You guessed correctly!`)}
+    {winOrLoseText.innerHTML =`You guessed ${suspect} and it was correct! :)`}
+    else 
+    //{console.log(`Your quess was wrong`)}
+    {winOrLoseText.innerHTML =`You guessed ${suspect} and it was wrong! :(`}
   // 1. Check if the suspect is the same as the secret person's name
   // 2. Set a Message to show in the win or lose section accordingly
   // 3. Show the win or lose section
   // 4. Hide the game board
+winOrLose.style.display = "flex"
+board.style.display = "none"
 }
```

Comment:
Later on, when you feel finished with this project you can remove the console.logs to clean up a bit :) Just a heads up, the opening curly bracket of an if/else statement should be on the same line:
```
if (suspect === secret.name) {
  //code
}
```
and the code inside curly brackets should be indented one level (one level per nesting level)